index.Rmd

```diff
@@ -5,6 +5,7 @@ author: Joe Prego
 ---
 
 # Introduction
+
 This project will examine the differences in spatial distance between the African American and White populations of Talbot County, Maryland during the second half of the 19th century based upon the population listed in the Lake, Griffing and Stevenson 1877 map, “An illustrated atlas of Talbot and Dorchester Counties, Maryland” and the census data from Talbot County in 1880. The Easton Market, located in Easton within greater Talbot County, was the main place to get economic resources during this period. This project will compare the travel distances between the African American and White populations to this market to test for differences in their costs of travel to obtain economic resources. These distances will be In doing so, it will as the questions, did the African American and White populations of Talbot County, Maryland have similar average costs of travel from their dwellings to the Easton town market to obtain economic resources during the later half of the 19th century?
```

Comment:
Good and clear intro. There are just a couple of grammatical errors, it looks like you started a sentence and didn't finish it ("These distance will be In doing so"). I would also maybe add a sentence or two justifying why you're studying this and why you might think they would be similar or different.